cicd-utils/info.json

```diff
@@ -17,8 +17,9 @@
       "operation": "unzip_export_template",
       "annotation": "unzip_export_template",
       "category": "utilities",
-      "description": "Unzip the specified downloaded zip export template",
+      "description": "Unzip the specified downloaded export template zip file",
```

Comment:
Suggestion: Unzip the specified export template zip

Reply:
done